sudoku/validate_sudoku.py

```diff
@@ -0,0 +1,119 @@
+# validate sudoku
+# input is a board
+# Return True if board is correct
+# Return False if board is incorrect
+
+
+def validate(board):
+
+    board_len = len(board)
+
+    # # check row numbers for duplicates
```

Comment:
why do you have `# #`. Only one # is enough.

Reply:
Cleaned.